Fix compilation issues with jars

https://paste.ubuntu.com/p/Yj7xYWvp3r/

The paste will expire so I am posting the stracktrace here too 
```java
[22:21:23 ERROR]: Could not load 'plugins/pgm.jar' in folder 'plugins'
org.bukkit.plugin.InvalidPluginException: java.lang.NoClassDefFoundError: app/ashcon/intake/fluent/CommandGraph
        at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:133) ~[sportpaper.jar:git-SportPaper-"374af6d"]
        at org.bukkit.plugin.SimplePluginManager.loadPlugin(SimplePluginManager.java:333) ~[sportpaper.jar:git-SportPaper-"374af6d"]
        at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:256) [sportpaper.jar:git-SportPaper-"374af6d"]
        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.loadPlugins(CraftServer.java:238) [sportpaper.jar:git-SportPaper-"374af6d"]
        at net.minecraft.server.v1_8_R3.DedicatedServer.init(DedicatedServer.java:198) [sportpaper.jar:git-SportPaper-"374af6d"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:667) [sportpaper.jar:git-SportPaper-"374af6d"]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_222]
Caused by: java.lang.NoClassDefFoundError: app/ashcon/intake/fluent/CommandGraph
        at java.lang.Class.forName0(Native Method) ~[?:1.8.0_222]
        at java.lang.Class.forName(Class.java:348) ~[?:1.8.0_222]
        at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:65) ~[sportpaper.jar:git-SportPaper-"374af6d"]
        at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:129) ~[sportpaper.jar:git-SportPaper-"374af6d"]
        ... 6 more
Caused by: java.lang.ClassNotFoundException: app.ashcon.intake.fluent.CommandGraph
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382) ~[?:1.8.0_222]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:102) ~[sportpaper.jar:git-SportPaper-"374af6d"]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:87) ~[sportpaper.jar:git-SportPaper-"374af6d"]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_222]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_222]
        at java.lang.Class.forName0(Native Method) ~[?:1.8.0_222]
        at java.lang.Class.forName(Class.java:348) ~[?:1.8.0_222]
        at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:65) ~[sportpaper.jar:git-SportPaper-"374af6d"]
        at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:129) ~[sportpaper.jar:git-SportPaper-"374af6d"]
        ... 6 more
```

--- a/docs/RUNNING.md
+++ b/docs/RUNNING.md
@@ -13,14 +13,14 @@ cd /path/to/folder
 
 2. Download the latest version of [SportPaper](https://github.com/Electroid/SportPaper), a fork of the Minecraft 1.8 server.
 ```bash
-curl https://pkg.ashcon.app/sportpaper -o sportpaper.jar
-curl https://pkg.ashcon.app/sportpaper-config -o sportpaper.yml
+curl https://pkg.ashcon.app/sportpaper -Lo sportpaper.jar
+curl https://pkg.ashcon.app/sportpaper-config -Lo sportpaper.yml
 ```
 
 3. Create a plugins folder and download the latest version of PGM.
 ```bash
 mkdir plugins
-curl https://pkg.ashcon.app/pgm -o plugins/pgm.jar
+curl https://pkg.ashcon.app/pgm -Lo plugins/pgm.jar
 ```
 
 4. Add your maps to the maps folder, if you don't have any, skip this step.

--- a/pom.xml
+++ b/pom.xml
@@ -25,24 +25,13 @@
             <groupId>app.ashcon</groupId>
             <artifactId>sportpaper</artifactId>
             <version>1.8.8-R0.1-SNAPSHOT</version>
+            <scope>provided</scope>
         </dependency>
         <!-- Command framework for developing simple commands -->
         <dependency>
             <groupId>app.ashcon.intake</groupId>
             <artifactId>intake-bukkit</artifactId>
             <version>1.1-SNAPSHOT</version>
-        </dependency>
-        <!-- Guava libraries that *must* match version used in SportPaper -->
-        <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-            <version>17.0</version>
-        </dependency>
-        <!-- YAML parsing library that *must* match version used in SportPaper -->
-        <dependency>
-            <groupId>org.yaml</groupId>
-            <artifactId>snakeyaml</artifactId>
-            <version>1.15</version>
         </dependency>
         <!-- Time convience libraries for "Instant" and "Duration" -->
         <!-- TODO: Use Java 8's standard time libraries -->
@@ -139,14 +128,9 @@
                     <minimizeJar>true</minimizeJar>
                     <artifactSet>
                         <excludes>
-                            <exclude>app.ashcon:sportpaper:jar:*</exclude>
                             <exclude>app.ashcon.intake:intake-core:jar:*</exclude>
                             <exclude>com.google.code.findbugs:jsr305:jar:*</exclude>
                         </excludes>
-                        <includes>
-                            <!-- Raven depends on this and does not shade it, so we have to -->
-                            <include>com.fasterxml.jackson.core:jackson-core</include>
-                        </includes>
                     </artifactSet>
                 </configuration>
                 <executions>

